django_jalali/db/models.py

```diff
@@ -49,16 +49,23 @@ class jDateField(models.DateField):
         'invalid_date': _('Invalid date: %s'),
     }
 
-    def __init__(self, verbose_name=None, name=None, auto_now=False,
-                 auto_now_add=False, **kwargs):
+    def __init__(
+        self, verbose_name=None, name=None, auto_now=False,
+        auto_now_add=False, default=models.NOT_PROVIDED, **kwargs
+    ):
 
         self.auto_now, self.auto_now_add = auto_now, auto_now_add
         # HACKs : auto_now_add/auto_now should be
         # done as a default or a pre_save.
         if auto_now or auto_now_add:
             kwargs['editable'] = False
             kwargs['blank'] = True
-        models.Field.__init__(self, verbose_name, name, **kwargs)
+
+        # Convert jdatetime.date default value to datetime.date
+        if isinstance(default, jdatetime.date):
+            default = datetime.date(default.year, default.month, default.day)
```

Comment:
hmm not sure how does this work. if the `default` is instance of e.g `jdatetime.date(1390,1,1)`, how do we use it in `datetime.date`? Does it evaluate to `datetime.date(1390,1,1)`

Reply:
Yes, I haven't considered it. I think we would have a problem with the `default` values like `jdatetime.date(1390, 6, 31)`.
What would be the best solution to fix this problem :thinking: ?